map.go

```diff
@@ -1809,3 +1816,26 @@ func sliceLen(slice any) (int, error) {
 	}
 	return sliceValue.Len(), nil
 }
+
+// NewBloomFilter creates a new bloom filter map.
+//
+// valueSize is the size of the values to be hashed.
+// maxEntries is used to approximate the size of the bloom filter bitmap.
+// numHashes specifies the number of hash functions to use (1-15).
+// If numHashes is 0, the default of 5 hash functions will be used.
+//
+// The bloom filter map type is available from Linux 5.16.
+func NewBloomFilter(name string, valueSize, maxEntries uint32, numHashes uint8) (*MapSpec, error) {
```

Comment:
We do not add constructors for specific map types in cilium/ebpf, its additional API surface area which is frankly not required since all fields on MapSpec are public.

Please do away with this bit.